tests/test_e2e_conversion.py

```diff
@@ -33,7 +35,8 @@ def get_converter():
     converter = DocumentConverter(
         format_options={
             InputFormat.PDF: PdfFormatOption(
-                pipeline_options=pipeline_options, backend=DoclingParseDocumentBackend
+                pipeline_options=pipeline_options,
+                backend=DoclingParseDocumentBackend,
```

Comment:
This should *not* be set, right?

Reply:
We decided we are setting the old v1 backend everywhere again, such that the test output won't change in this PR. We make another PR where we remove this `backend=DoclingParseDocumentBackend` in all places, such that we have a separate PR which holds the test diff for V4.